lib/libyear_bundler/query.rb

```diff
@@ -16,35 +16,11 @@ def initialize(gemfile_path)
     end
 
     def execute
-      gems = []
-      bundle_outdated.lines.each do |line|
+      bundle_outdated.lines.each_with_object([]) do |line, gems|
         match = BOP_FMT.match(line)
         next if match.nil?
-        gems.push(::Models::Gem.new(match))
+        gems.push(::LibyearBundler::Models::Gem.new(match))
       end
-      gems.each do |gem|
-        if @argv.include?("--versions") || @argv.include?("--all")
-          gem.version_number_delta =
-            ::Calculators::VersionNumberDelta.calculate(
-              gem.installed_version,
-              gem.newest_version
-            )
-        end
-
-        if @argv.include?("--releases") || @argv.include?("--all")
-          gem.version_sequence_delta =
-            ::Calculators::VersionSequenceDelta.calculate(
-              gem.installed_version,
-              gem.newest_version
-            )
-        end
-
-        gem.libyears = ::Calculators::Libyear.calculate(
-          gem.installed_version_release_date,
-          gem.newest_version_release_date
-        )
-      end
-      gems
```

Comment:
Maybe we should call this class `BundleOutdated` since that's all it does now.

Reply:
It does the parsing, which is significant. Maybe `BundleOutdatedParser`. Too verbose?

Reply:
> It does the parsing, which is significant. Maybe BundleOutdatedParser. Too verbose?

I think that would be misleading, because it doesn't *only* do parsing, it also runs the command. I prefer `BundleOutdated`.